peopleops/performance-review.md

```diff
@@ -0,0 +1,13 @@
+---
+navTitle: Performance review
+---
+
+# Performance Review
+
+In September a performance adjustment is done for all employees. The performance
+review is based on [our values](../company#values), and meeting role
+expectations. A review is held by the direct manager of each employeee. The
+manager will write down their expectations, where these were met and where the
+employee can still grow. This text is shared with each individual employee.
+
+Performance as review might influence the [compensation adjustment](../peopleops/compensation#adjustment)
```

Comment:
```suggestion
In September a performance review is done for all employees. The performance
review is based on [our values](../company#values), and meeting role
expectations. A review is held by the direct manager of each employee. The
manager will write down their expectations, where these were met and where the
employee can still grow. This text is shared with each individual employee.

The performance review will be used as a factor towards any [compensation adjustment](../peopleops/compensation#adjustment) made.
```

Do we want to consider documenting a minimum time with the company when considering how this may influence any compensation changes? Either here or the compensation section?

Reply:
> The performance review will be used as a factor towards any

I'd like to keep that at 'might', as compensation that's increasing due to other factors but performance wasn't great now implies performance was fine.

Reply:
I have changed the proposal back to include 'might'.